play-server-circe/src/main/scala/endpoints/play/routing/CirceEntities.scala

```diff
@@ -1,14 +1,17 @@
-package endpoints
+package endpoints.play.routing
 
+import endpoints.algebra
+import endpoints.algebra.CirceEntities.CirceCodec
+import endpoints.play
 import io.circe.jawn
-import play.api.libs.concurrent.Execution.Implicits.defaultContext
-import play.api.mvc.{BodyParsers, Results}
-import PlayCirce.circeJsonWriteable
+import _root_.play.api.libs.concurrent.Execution.Implicits.defaultContext
+import _root_.play.api.mvc.{BodyParsers, Results}
```

Comment:
Would it be possible to avoid importing `endpoints.play`, above, so that we don’t need to use the `_root_` package name?